src/request/process-request-options.ts

```diff
@@ -0,0 +1,213 @@
+import { OutgoingHttpHeaders } from 'http';
+import {
+    castArray,
+    isArrayBuffer,
+    isBuffer,
+    isObject,
+    isUndefined,
+} from 'lodash';
+import isStream from 'is-stream';
+import {
+    AuthOptions,
+    Credentials,
+    ExternalRequestOptions,
+    Params,
+} from './interfaces';
+import {
+    RequestOptions,
+    generateUniqueId,
+} from 'testcafe-hammerhead';
+import TestRun from '../test-run';
+import CONTENT_TYPES from '../assets/content-types';
+import HTTP_HEADERS from '../utils/http-headers';
+import { RUNTIME_ERRORS } from '../errors/types';
+import { APIError } from '../errors/runtime';
+import { GetAjaxProxyUrlCommand } from '../test-run/commands/actions';
+
+const DEFAULT_ACCEPT            = { [HTTP_HEADERS.accept]: `${CONTENT_TYPES.json}, ${CONTENT_TYPES.textPlain}, ${CONTENT_TYPES.all}` };
+const DEFAULT_IS_REQUEST        = { [HTTP_HEADERS.isRequest]: true };
+const METHODS_WITH_CONTENT_TYPE = ['post', 'put', 'patch'];
+const DEFAULT_REQUEST_TIMEOUT   = 2 * 60 * 1000;
```

Comment:
It's not necessary. The default values are in the [hammerhead](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/request-pipeline/destination-request/default-request-timeout.ts) module.

Reply:
In the hammerhead, it is used only for creating sessions. I added using these values also for creating RequesOptions